src/olympia/devhub/urls.py

```diff
@@ -74,6 +74,8 @@
     url('^versions$', views.version_list, name='devhub.addons.versions'),
     url('^versions/delete$', views.version_delete,
         name='devhub.versions.delete'),
+    url('^versions/undisable$', views.version_undisable,
```

Comment:
Maybe call this `restore` instead? Nitpick though, I've never seen `undisabled` as a word before, I learned something new :)